templates/Segment_Image_Data.ipynb

```diff
@@ -17,10 +17,9 @@
    "source": [
```

Comment:
<p>Clear this cell's output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>